Bottom: 67f2c31cf7b91cd7733879bb0c8ffeed3d374626
Top:    6abde974ccca532794f5d6e4b7c6cb6cc8c2ee8a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-03 17:24:07 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 890e03c..3ab77c6 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -110,6 +110,7 @@ hidgl_init_triangle_array (triangle_buffer *buffer)
   glGenBuffers (1, &buffer->vbo_id);
   glBindBuffer (GL_ARRAY_BUFFER, buffer->vbo_id);
 
+  buffer->triangle_array = NULL;
   buffer->local = false;
   hidgl_reset_triangle_array (buffer);
 }
